memlab.bib

```diff
@@ -5137,7 +5229,7 @@ @article{HeusEtal16
 	Title = {Episodic sequence memory is supported by a theta-gamma phase code},
 	Volume = {Advance online publication. doi:10.1038/nn.4374},
 	Year = {2016},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAYLi4vRG93bmxvYWRzL25uLjQzNzQucmlzTxEBXgAAAAABXgACAAAETWFpbgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA085gjkgrAAAACSXoC25uLjQzNzQucmlzAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABBJofT8L8pAAAAAAAAAAAAAQACAAAJIAAAAAAAAAAAAAAAAAAAAAlEb3dubG9hZHMAABAACAAA086YzgAAABEACAAA0/D3aQAAAAEADAAJJegABrCLAAYwUwACACxNYWluOlVzZXJzOgBqbWFubmluZzoARG93bmxvYWRzOgBubi40Mzc0LnJpcwAOABgACwBuAG4ALgA0ADMANwA0AC4AcgBpAHMADwAKAAQATQBhAGkAbgASACRVc2Vycy9qbWFubmluZy9Eb3dubG9hZHMvbm4uNDM3NC5yaXMAEwABLwAAFQACAA///wAAAAgADQAaACQAPwAAAAAAAAIBAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAGh}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QGC4uL0Rvd25sb2Fkcy9ubi40Mzc0LnJpc9IXCxgZV05TLmRhdGFPEQFeAAAAAAFeAAIAAARNYWluAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADTzmCOSCsAAAAJJegLbm4uNDM3NC5yaXMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEEmh9PwvykAAAAAAAAAAAABAAIAAAkgAAAAAAAAAAAAAAAAAAAACURvd25sb2FkcwAAEAAIAADTzpjOAAAAEQAIAADT8PdpAAAAAQAMAAkl6AAGsIsABjBTAAIALE1haW46VXNlcnM6AGptYW5uaW5nOgBEb3dubG9hZHM6AG5uLjQzNzQucmlzAA4AGAALAG4AbgAuADQAMwA3ADQALgByAGkAcwAPAAoABABNAGEAaQBuABIAJFVzZXJzL2ptYW5uaW5nL0Rvd25sb2Fkcy9ubi40Mzc0LnJpcwATAAEvAAAVAAIAD///AACABtIbHB0eWiRjbGFzc25hbWVYJGNsYXNzZXNdTlNNdXRhYmxlRGF0YaMdHyBWTlNEYXRhWE5TT2JqZWN00hscIiNcTlNEaWN0aW9uYXJ5oiIgXxAPTlNLZXllZEFyY2hpdmVy0SYnVHJvb3SAAQAIABEAGgAjAC0AMgA3AEAARgBNAFUAYABnAGoAbABuAHEAcwB1AHcAhACOAKkArgC2AhgCGgIfAioCMwJBAkUCTAJVAloCZwJqAnwCfwKEAAAAAAAAAgEAAAAAAAAAKAAAAAAAAAAAAAAAAAAAAoY=}}
```

Comment:
Not sure what this jumble is...but it looks like an error in the old version, and should be removed from the new version...

Reply:
Looks like this entry was added via a .ris file. I've removed the file.